guides/doc-Managing_Content/master.adoc

```diff
@@ -9,6 +9,11 @@ include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[le
 endif::[]
 
 // Render only for relevant and finished contexts
+ifndef::katello,satellite,orcharhino[]
+Content management requires the Katello plugin.
+Conversion of an existing installation is not supported.
+See the {InstallingServerDocURL}[Katello installation guide] on how to install Katello.
```

Comment:
```suggestion
Content management requires the Katello plug-in.
You cannot install the Katello plug-in on your existing {ProjectServer}.
For more information, see {InstallingServerDocURL}{InstallingServerDocTitle}.
```

Reply:
I explicitly didn't use `{InstallingServerDocTitle}` because it's for the current flavor, not the Katello flavor.

Reply:
But if the link resolves for "foreman-el", users will end up at "Installing Foreman on EL", not katello.

Reply:
Oh yes, you're right.

Reply:
Now you've distracted me with unrelated work: https://github.com/theforeman/foreman-documentation/pull/2341

Reply:
Updated now with an explicit link & comment.